src/deepqmc/hkext.py

```diff
@@ -99,3 +100,28 @@ def __call__(self, inputs):
             if i < (n_layers - 1) or not self.last_linear:
                 out = self.activation(out)
         return out + inputs if self.residual else out
+
+
+class ResidualConnection:
```

Comment:
I think this is a bit more flexible than including the option for residual connection in MLP. Also we can unify the one and two particle stream residuals like this.

Reply:
Ah indeed the `residual` `kwarg` of MLP still exists and should be removed. Also, the shared u network should not have `residual` `kwarg` passed to it as this is now handled outside the MLP. I'll make these changes. Is there anywhere else where the `residual` `kwarg` of MLP is used?

Reply:
I think that was the only remaining use of the residual argument.